src/data_io.py

```diff
@@ -64,7 +66,7 @@ def save_data(
         height = self.metadata.height
 
         zarr_store = zarr.open(
-            store=f"{save_directory / save_name}.zarr",
+            store=f"{data_directory / data_name}.zarr",
```

Comment:
sort of strikes me as odd, i would expect to be given a whole path here? but not sure what da goal is

Reply:
Oh, cause there are going to be multiple stages of saved files, all in the same directory. So I wanted to compartmentalize those. (dir/stage1.zarr, dir/stage2.zarr, etc.)